hazelcast/src/test/java/com/hazelcast/executor/ExecutorServiceTest.java

```diff
@@ -80,34 +69,188 @@
 @Category(QuickTest.class)
 public class ExecutorServiceTest extends ExecutorServiceTestSupport {
 
```

Comment:
Probably it would be better (in the future) to set it up in the same way I do for other test. 

AtomicLongBasicTest
AtomicLongBasicDistributedTest
AtomicLongBasicLocalTest

This way you know that all operations work in a local and remote setting.